src/server/gc_worker/mod.rs

```diff
@@ -137,7 +77,7 @@ impl<T: PdClient + 'static> GcSafePointProvider for Arc<T> {
     }
 }
 
-enum GcTask {
+pub(self) enum GcTask {
```

Comment:
I've never seen this... What's its purpose?

Reply:
Trying to export to other submodules but do not to export externally. This is removed in the next commit. PTAL again.

Reply:
It sounds weird. I think it should be naturally visible by submodules...